src/core/dfn-panel.js

```diff
@@ -111,16 +111,14 @@ function referencesToHTML(id, links) {
     );
   };
 
-  /**
-   * @param {[string, string[]]} entry
-   * @returns {HTMLLIElement}
-   */
-  const listItemToHTML = entry =>
-    hyperHTML`<li>${toLinkProps(entry).map(
-      link => hyperHTML`<a href="#${link.id}">${link.title}</a>${" "}`
-    )}</li>`;
+  const listItems = [...titleToIDs].map(
+    entry =>
+      hyperHTML`<li>${toLinkProps(entry).map(
+        link => hyperHTML`<a href="#${link.id}">${link.title}</a>${" "}`
+      )}</li>`
+  );
 
-  return hyperHTML`<ul>${[...titleToIDs.entries()].map(listItemToHTML)}</ul>`;
+  return hyperHTML`<ul>${listItems}</ul>`;
```

Comment:
Can we skip this change? I personally prefer the previous version. Not a big deal though.

Reply:
I also don't have a preference here. I'm happy with either.

Reply:
I prefer to avoid complex expression within template strings 👀

Should I keep the function separate, if that is what you prefer?

Reply:
Separate `listItemToHtml` would be great